installation-manager/src/main/java/org/wildfly/core/instmgr/logging/InstMgrLogger.java

```diff
@@ -36,76 +36,76 @@
     OperationFailedException serverAlreadyPrepared();
 
-    @Message(id = 5, value = "Invalid status change found for the artifact: '%s'")
+    @Message(id = 3, value = "Invalid status change found for the artifact: '%s'")
     RuntimeException unexpectedArtifactChange(String artifact);
 
-    @Message(id = 6, value = "Invalid status change found for the configuration change: '%s'")
+    @Message(id = 4, value = "Invalid status change found for the configuration change: '%s'")
     RuntimeException unexpectedConfigurationChange(String channel);
 
-    @Message(id = 7, value = "Channel name is mandatory")
+    @Message(id = 5, value = "Channel name is mandatory.")
     OperationFailedException missingChannelName();
 
-    @Message(id = 8, value = "No repositories are defined in the '%s' channel.")
+    @Message(id = 6, value = "No repositories have been defined in the '%s' channel.")
     OperationFailedException noChannelRepositoriesDefined(String channelName);
 
-    @Message(id = 9, value = "The '%s' channel's repository does not have any defined URL.")
+    @Message(id = 7, value = "The '%s' repository in the channel does not have its URL defined.")
     OperationFailedException noChannelRepositoryURLDefined(String channelName);
 
-    @Message(id = 10, value = "The repository URL '%s' for '%s' channel is invalid.")
+    @Message(id = 8, value = "The repository URL '%s' for '%s' channel is invalid.")
     OperationFailedException invalidChannelRepositoryURL(String repoUrl, String channelName);
 
-    @Message(id = 11, value = "The '%s' channel's repository does not have any defined ID")
+    @Message(id = 9, value = "The '%s' repository in the channel does not have its ID defined.")
     OperationFailedException noChannelRepositoryIDDefined(String channelName);
 
-    @Message(id = 12, value = "The GAV manifest '%s' for '%s' channel is invalid.")
+    @Message(id = 10, value = "The manifest GAV coordinate '%s' for '%s' channel is invalid.")
     OperationFailedException invalidChannelManifestGAV(String gav, String channelName);
 
-    @Message(id = 13, value = "The URL manifest '%s' for '%s' channel is invalid.")
+    @Message(id = 11, value = "The manifest URL '%s' for '%s' channel is invalid.")
     OperationFailedException invalidChannelManifestURL(String url, String channelName);
 
-    @Message(id = 14, value = "'local-cache' cannot be used when 'no-resolve-local-cache' is enabled.")
+    @Message(id = 12, value = "You cannot use 'local-cache' option when 'no-resolve-local-cache' is enabled.")
     OperationFailedException localCacheWithNoResolveLocalCache();
 
-    @Message(id = 15, value = "You cannot use 'maven-repo-file' in conjunction with 'repositories' because they are mutually exclusive.")
+    @Message(id = 13, value = "You cannot use 'maven-repo-file' option with 'repositories' option because they are mutually exclusive.")
```

Comment:
s/cannot use/cannot use the/g

Reply:
Assume this same comment applies to any other similar cases.

Reply:
Done.